package/MDAnalysis/coordinates/FHIAIMS.py

```diff
@@ -0,0 +1,261 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+FHIAIMS file format --- :mod:`MDAnalysis.coordinates.FHIAIMS`
+=============================================================
+
+Classes to read and write _FHIAIMS coordinate files.
+
+Classes
+-------
+
+.. autoclass:: Timestep
+   :members:
+
+.. autoclass:: FHIAIMSReader
+   :members:
+
+.. autoclass:: FHIAIMSWriter
+   :members:
+
+
+Developer notes: ``FHIAIMSWriter`` format strings
+-------------------------------------------------
+
+The :class:`FHIAIMSWriter` class has a :attr:`FHIAIMSWriter.fmt` attribute, which is a dictionary of different
+strings for writing lines in ``.gro`` files.  These are as follows:
+
+``xyz``
+  An atom line without velocities.  Requires that the 'name' and 'pos' keys be supplied.
+  E.g.::
+
+     fmt['xyz'].format(pos=(0.0, 1.0, 2.0), name='O')
+
+``box_orthorhombic``
+  The final line of the gro file which gives box dimensions.  Requires
+  the box keyword to be given, which should be the three cartesian dimensions.
+  E.g.::
+
+     fmt['box_orthorhombic'].format(box=(10.0, 10.0, 10.0))
+
+``box_triclinic``
+  The (optional) initial lines of the file which gives box dimensions. 
+  Requires the box keyword, as a length 9 vector.  This is a flattened
+  version of the (3,3) triclinic vector representation of the unit
+  cell.  
+
+.. _FHIAIMS: https://aimsclub.fhi-berlin.mpg.de/
```

Comment:
Add definition of the file format. Documentation included in MDAnalysis also falls under GPL so if you copy and paste from other sources, tell us source and license of those docs. If you write it in your own words then there shouldn't be a problem.

Reply:
added my own description